src/IPC/ClusterIPC.ts

```diff
@@ -33,7 +33,7 @@ export class ClusterIPC extends EventEmitter {
 	}
 
 	public async init() {
-		this.nodeSocket = await this.node.connectTo('Master', this.socket);
+		this.nodeSocket = await this.node.connectTo('Master', String(this.socket));
```

Comment:
This can also be a `number`, the `String()` cast is unneeded.

https://github.com/kyranet/veza/blob/b2d8c3ebc785e2a57aedb9226c058852417408f1/typings/index.d.ts#L54-L55

Reply:
I would agree but it displayed an error thus it was fixed here and https://github.com/Dev-Yukine/Kurasuta/commit/27128eedb223adb1a3372da9249168cd2c3e3267 here.

Reply:
What error?

Reply:
It can be seen here https://imgur.com/FoZRjMc